ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2384,6 +2402,76 @@ class PartCentredForceHolder {
     planetarium_camera_adjuster_.should_transfer_camera_coordinates = true;
   }
 
+  private void ParseDrawStyle(
```

Comment:
Move this function to https://github.com/mockingbirdnest/Principia/blob/master/ksp_plugin_adapter/config_node_parsers.cs, and make it an extension method (`this ConfigNode parameters`).

Reply:
I have moved these as requested, and renamed to fit the existing style.